### PR TITLE
Add cleanup routine

### LINKED ISSUE #7: Add cleanup  routine
Clean up processed files and send status update.

### PR DESCRIPTION
**Pull request by bonniegee**
Fixes #7 

The setup for the tests for this routine overlap with functionality in other services, but I figured that should be looked at in #22 and #24.

(Note: this is branched from #18, which should be merged first)

### CODE REVIEW

File: create_derivatives/tests.py [hunk 1]
```diff
@@ -164,3 +164,34 @@ def test_run(self, mock_upload_files, mock_init):
...
+                bag_path=bag_path,
+                origin="digitization",
+                as_data="sdjfkldsjf",
+                dimes_identifier=bag,
+                process_status=Bag.UPLOADED)
+
+    def test_run(self):
+        cleaned_up = CleanupRoutine().run()
+        self.assertEqual(len(list(Path(settings.TMP_DIR).glob('*'))), 0)
+        self.assertEqual(cleaned_up[0], "Bags successfully cleaned up.")
```
Comment (helrond) [inline create_derivatives/tests.py]:
```suggestion
        msg, object_list = CleanupRoutine().run()
        self.assertEqual(len(list(Path(settings.TMP_DIR).glob('*'))), 0)
        self.assertEqual(message, "Bags successfully cleaned up.")
        self.assertTrue(isinstance(object_list), list)
        self.assertEqual(len(object_list), 1)
```
Adding a couple more asserts.

**Review by helrond — CHANGES_REQUESTED** [review]
Could add a few more asserts for a stronger test.

**Review by helrond — APPROVED** [review]

File: create_derivatives/tests.py [hunk 2]
```diff
@@ -199,3 +199,36 @@ def test_run(self, mock_upload_files, mock_init):
...
+        bag_path = str(Path(settings.TMP_DIR, bag))
+        if not Path(bag_path).exists():
+            shutil.copytree(Path("create_derivatives", "fixtures", fixture_directory, bag), bag_path)
+            Bag.objects.create(
+                bag_identifier="sdfjldskj",
+                bag_path=bag_path,
+                origin="digitization",
+                as_data="sdjfkldsjf",
+                dimes_identifier=bag,
+                process_status=Bag.UPLOADED)
```
Comment (p-galligan) [inline create_derivatives/tests.py:224]: Was there a decision on why we're not using fixtures for this functionality like for other tests?

File: create_derivatives/routines.py [hunk 3]
```diff
@@ -156,4 +157,12 @@ def run(self, replace):
 
 
 class CleanupRoutine:
```
Comment (HaSistrunk) [inline create_derivatives/routines.py:159]: Does "cleanup" by definition mean to remove stuff? I was going to suggest being explicit in a docstring and/or the `msg` about what cleanup meant, but maybe that doesn't need defining?

Comment (bonniegee) [reply create_derivatives/routines.py:159]: You're right that a docstring would be helpful. In the context of seeing this message in the wild (e.g., in Zodiac) I think it's sufficient, but I also don't have particularly strong feelings about what the message says.

**Review by HaSistrunk — APPROVED** [review]

**Review by p-galligan — CHANGES_REQUESTED** [review]

**Review by p-galligan — APPROVED** [review]

**Pull request merged**